raspbian/applications/camera.md

```diff
@@ -448,7 +448,9 @@ Outputs debugging/information messages during the program run.
 	--timeout,	-t		Time before the camera takes picture and shuts down
 ```
 
-The program will run for this length of time, then take the capture (if output is specified). If not specified, this is set to 5 seconds.
+The program will run for this length of time, in milliseconds, then take the capture (if output is specified). If not specified, this is set to 5 seconds (-t 5000). Note that low values (less than 500ms, although it does depend on other settings) may not give enough time for the camera to start up and provide enough frames for the automatic algorithms like AWB and AGC to provide accurate results.
```

Comment:
I wonder if the first sentence ending with "... (if output is specified)" while the second sentence starts with "If not specified, ..." might potentially cause confusion? :shrug:  Would changing the second sentence to start with "If the timeout is not specified, ..." be better?

Maybe "it does depend on other settings" sounds better as "it also depends on other settings" ?

Reply:
Have changed it around a bit.